build/scritps/valgrind.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
What is this script is used for?

Reply:
This is used primarily for Callgrind invocation, and in general in any case you'd like to run valgrind without RLTest.